daft/dataframe/dataframe.py

```diff
@@ -1178,6 +1178,27 @@ def _create_metadata_param(metadata: Optional[Dict[str, str]]):
 
         return with_operations
 
+    @DataframePublicAPI
+    def write(self, sink: "DataSink[T, R]", **kwargs) -> "R":
```

Comment:
```suggestion
    def write_sink(self, sink: "DataSink[T, R]", **kwargs) -> "R":
```

Reply:
Also all args should be encapsulated into the __init__ of the WriteSink

Reply:
Done